force-app/main/default/lwc/primaryAffiliationsModalBody/primaryAffiliationsModalBody.js

```diff
@@ -115,6 +117,11 @@ export default class PrimaryAffiliationsModalBody extends LightningElement {
     }
 
     get affiliationsDesc() {
-        return this.labelReference.modalBodyEditSave + " " + this.affiliationsHyperLink;
+        if (this.affiliationsAction === "edit") {
```

Comment:
We probably want to use a switch statement here given that we will have a third option. This is functional so I am going to approve it.